include/boost/beast/core/detail/is_invocable.hpp

```diff
@@ -51,6 +53,26 @@ struct is_invocable<C, R(A...)>
 };
 /** @} */
 
+template<class CompletionToken, class Signature, class = void>
+struct is_completion_token_for : std::false_type
+{
+};
+
+struct dummy_initiation
```

Comment:
I avoid using the word "dummy" in my code

Reply:
alternative?

Reply:
`any_initiation`